<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalPropertyWidget</class>
 <widget class="QWidget" name="GlobalPropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="familyLabel">
     <property name="text">
      <string>Family:</string>
     </property>
     <property name="buddy">
      <cstring>familySelector</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="TagFamilyComboBox" name="familySelector"/>
   </item>
   <item row="0" column="3">
    <widget class="QDoubleSpinBox" name="tagSizeEdit">
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="authorLabel">
     <property name="text">
      <string>Author:</string>
     </property>
     <property name="buddy">
      <cstring>authorEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="buddy">
      <cstring>nameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="tagSizeLabel">
     <property name="text">
      <string>Tag Size (mm):</string>
     </property>
     <property name="buddy">
      <cstring>tagSizeEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="commentEdit"/>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="authorEdit"/>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QLabel" name="commentLabel">
     <property name="text">
      <string>Comments:</string>
     </property>
     <property name="buddy">
      <cstring>commentEdit</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TagFamilyComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">fort-studio/widget/base/TagFamilyComboBox.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>familySelector</tabstop>
  <tabstop>tagSizeEdit</tabstop>
  <tabstop>nameEdit</tabstop>
  <tabstop>authorEdit</tabstop>
  <tabstop>commentEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
